packages/tradershub/src/features/cfd/screens/TradeScreen/TradeLink/TradeLink.tsx

```diff
@@ -43,17 +43,17 @@ const TradeLink = ({ app = 'linux', platform, webtraderUrl = '' }: TTradeLinkPro
     return (
         <div className='flex items-center justify-between px-16 py-24 border-solid border-t-1 border-system-light-secondary-background'>
             <div className='flex items-center gap-16'>
-                {(platform === mt5Platform || app === ctraderPlatform) && (
+                {(platform === mt5Platform || app === ctraderPlatform || app === 'ctrader_ios') && (
```

Comment:
since we need to check for lots of conditions, let's split this out to different components, cTraderTraderLink, ...